src/components/Combobox/components/ComboboxDropdown.js

```diff
@@ -75,6 +79,7 @@ const ComboboxDropdown = React.forwardRef((props, ref) => {
         </div>
       </FormElementControl>
       {renderListbox && renderListbox()}
+      {!isErrorHidden && <FormElementError error={error} id={`error-${id}`} />}
```

Comment:
I would argue that we should strip these props in everything but `Picklist` components. Otherwise, beneath the listbox seems like an odd place to render these